### DISCUSSION TITLE
Why doesn't symfony load the welcome screen?

### DISCUSSION THREAD

**Discussion by juliodenis91** [symfony/symfony#45904] [Q&A] [answered]
I create a project with the command: "composer create-project symfony/website-skeleton projectName", it just tells me: The connection has been reset The connection to the server was reset while the page was loaded. The site might be temporarily unavailable or too busy. Please try again in a few moments. If you can't load any pages, check your computer's network connection. If your computer or network is protected by a firewall or proxy, make sure Firefox has permission to access the web
![w4Fby](https://user-images.githubusercontent.com/30736134/161395678-b6c7d3ce-39ab-4031-85c1-e3264acb3691.png)

I am using: xampp 3.3.0, symfony cli 5.4.2, composer 2.3.2 and I have configured my virtual host too
Note: it is a newly install

[ed](url) project via composer and it does not load the welcome screen

**Comment by fd6130** [comment, marked as answer]
Since you have symfony cli, how about try serve it using `symfony serve --no-tls` in your project instead?

**Reply by juliodenis91** [reply]
I solved the problem in the following way:
In httpd.conf file, on Windows
 
Add this code at the end:
![imagen](https://user-images.githubusercontent.com/30736134/165957581-fb2fc6cb-929f-43a7-9358-d53825853e5e.png)
 
and ready !!!
thanks for answering, have a nice day !!!
